build.gradle.kts

```diff
@@ -14,7 +14,7 @@ object Versions {
     const val kafkaVersion = "2.3.1"
     const val kafkaEmbeddedVersion = "2.4.0"
     const val kluentVersion = "1.61"
-    const val ktorVersion = "1.4.3"
+    const val ktorVersion = "1.5.0"
```

Comment:
Finnes en 1.5.2-versjon også, men 1.5.0 skal være tilstrekkelig for å løse den rare feilen.